common/perl/get_unique_updates/get_unique_updates.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+
+# ABSTRACT: Find and return all unique package names in input file with highest version string
+# PODNAME: get_unique_updates.pl
+
+use Modern::Perl;
```

Comment:
Being an infrequent perl user, I got an error message when trying to run this script after checking it out with `gh pr checkout 1622`:

```
$ ./get_unique_updates.pl /tmp/solus_builds_2024-02-04.md 
Can't locate Modern/Perl.pm in @INC (you may need to install the Modern::Perl module) (@INC entries checked: /usr/lib64/perl5/site_perl/5.38/x86_64-linux-thread-multi /usr/lib64/perl5/site_perl/5.38 /usr/lib64/perl5/vendor_perl/5.38/x86_64-linux-thread-multi /usr/lib64/perl5/vendor_perl/5.38 /usr/lib64/perl5/5.38/x86_64-linux-thread-multi /usr/lib64/perl5/5.38) at ./get_unique_updates.pl line 6.
BEGIN failed--compilation aborted at ./get_unique_updates.pl line 6.
```

I did a few `eopkg search`  invocations, but nothing useful immediately popped up?

It might be a good idea for the documention to state explicitly how to ensure this package/module is installed prior to running this script?

Reply:
I wrote a block to check for the needed modules, and provide instructions to the user on how to install if a module can't be found. It should now detect for you that Modern::Perl is not installed. Let me know if the instructions in the output work for you.

Reply:
I still get the same error as above and (sadly!) _no_ instructions on how to install it -- despite seeing in your commit that you did add instructions -- just not for getting `Modern::Perl` installed it seems...?